libcudacxx/include/cuda/std/detail/libcxx/include/__hash_table

```diff
@@ -22,14 +22,6 @@
 #include "__assert" // all public C++ headers provide the assertion handler
 #include "__debug"
```

Comment:
suggestion: have you intentionally not added system header here? Is it because there's no cuda mode for it? I'd rather still add the system header.

Reply:
Actually yeah, I decided to try and be more consistent and add it to all non exported headers too